src/classes/UserCart.ts

```diff
@@ -36,7 +36,12 @@ class UserCart extends Cart {
 
         // TODO: Check for dupes
 
-        if ((this.bot.handler as MyHandler).hasDupeCheckEnabled()) {
+        const isDupedCheckEnabled = (this.bot.handler as MyHandler).hasDupeCheckEnabled();
+        const keyPrice = this.bot.pricelist.getKeyPrice();
+        const theirItemsValue = this.getTheirCurrencies().toValue(keyPrice.metal);
+        const minimumKeysDupeCheck = (this.bot.handler as MyHandler).getMinimumKeysDupeCheck() * keyPrice.toValue();
+
+        if (isDupedCheckEnabled && theirItemsValue > minimumKeysDupeCheck) {
```

Comment:
And here you don't do the check for australiums / effects?

Reply:
Well yes, I should add that but I am not sure how.